.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,40 @@
+---
+labels: Needs Review
+name: Pull Request
+---
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+- [ ] Desktop screenshots
+- [ ] Mobile width screenshots
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Enhancement (improves on existing features/functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Version/configuration (updating dependency versions or modifying configuration settings)
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have added tests to cover my changes.
+- [ ] All new and existing tests passed.
+- [ ] My change includes database migrations
+- [ ] My change includes new configuration settings that need to be updated in RC/Production
```

Comment:
Should we have a box regarding that this configuration is already in place or at least the devops PR is staged (before merging)? I think generally that's a good idea, although I think even better is if we strive to feature flag our stuff more consistently based on the presence of configuration.

Reply:
My intent here is to make it visible that there are config changes. The assumption is that by highlighting it, that will remind the implementer or reviewer that the extra work is necessary. I'll bias toward _not_ adding extra buttons for now, unless there's strong sentiment about it.